code/modules/crew_objectives/medical_objectives.dm

```diff
@@ -1,14 +1,13 @@
 /*				MEDICAL OBJECTIVES				*/
 
 /datum/objective/crew/morgue //Ported from old Hippie
-	explanation_text = "Ensure there are no corpses on the station outside of the morgue when the shift ends."
+	explanation_text = "Ensure the Medbay has been cleaned of any corpses when the shift ends."
 	jobs = "chiefmedicalofficer,geneticist,medicaldoctor"
 
 /datum/objective/crew/morgue/check_completion()
 	for(var/mob/living/carbon/human/H in GLOB.mob_list)
-		if(H.stat == DEAD && (H.z in SSmapping.levels_by_trait(ZTRAIT_STATION)))
-			if(get_area(H) != /area/medical/morgue)
-				return FALSE
+		if(H.stat == DEAD && (get_area(H) == /area/medical/central))
```

Comment:
`if(H.stat == DEAD && ((get_area(H) == typesof(/area/medical/medbay/central)) || (get_area(H) == typesof(/area/medical/patients_rooms))`

Only covers the hallways and the patient rooms, which I think is fair. Last minute surgery is fairly common.

I hate how there's no global list just for dead crew's bodies, but that's out of the scope of this PR.

Reply:
I disagree. The objective should do what it says and cover all of medical

Reply:
Very reasonable. Is there an area for the cloning room? I'd like to add that also. Getting a really large if statement going, though.

Reply:
Making it cover all of medical makes it be less likely to break in the future if more medical areas are added on certain maps.

Reply:
Is morgue considered medical?

Reply:
It is but you could exclude that